src/main/java/org/jenkinsci/plugins/workflow/libs/ResourceStep.java

```diff
@@ -50,6 +53,14 @@ public String getResource() {
         return resource;
     }
 
+    public String getEncoding() {
+        return encoding;
+    }
+
+    @DataBoundSetter public void setEncoding(String encoding) {
```

Comment:
Would be helpful to have javadoc explaining handling of null & empty strings here.